crates/re_viewport/src/viewport_blueprint.rs

```diff
@@ -226,6 +226,30 @@ impl ViewportBlueprint {
         ctx.save_blueprint_component(&VIEWPORT_PATH.into(), component);
     }
 
+    /// Duplicates a space view and its entity property overrides
+    pub fn duplicate_space_view(
+        &self,
+        space_view_id: &SpaceViewId,
+        ctx: &ViewerContext<'_>,
+    ) -> Option<SpaceViewId> {
+        let Some(space_view) = self.space_view(space_view_id) else {
+            return None;
+        };
+
+        let new_space_view = space_view.duplicate();
+        let new_space_view_id = new_space_view.id;
+
+        // copy entity properties from the old space view
+        let data_result = space_view.root_data_result(ctx.store_context, ctx.blueprint_query);
+        let new_data_result =
+            new_space_view.root_data_result(ctx.store_context, ctx.blueprint_query);
+        new_data_result.save_override(data_result.individual_properties().cloned(), ctx);
```

Comment:
This is still only a portion of the data to be duplicated -- I think we also need to walk the entire query override tree and copy all of the data overrides as well.

Assuming you don't want to do that now since this incremental improvement unblocks Clement, feel free to just add a TODO(#4977) and just fully close out the issue (which I updated) to mention the other data.

Reply:
I will take the suggested shortcut, thanks :)